.github/workflows/build.yaml

```diff
@@ -200,9 +200,87 @@ jobs:
       working-directory: ${{ github.workspace }}/SymStore
       run: aws --endpoint-url https://rgw.ctrl-c.liu.se s3 sync --size-only --exclude * --include *.dl_ --include *.pd_ . s3://openmw-sym
 
+  dynamic-linux:
+    strategy:
+      fail-fast: true
+
+    runs-on: ubuntu-24.04
+
+    name: dynamic-linux
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - run: sudo apt update
+
+    - run: >
+        sudo apt install -y
+        autoconf-archive
+        libegl1-mesa-dev
+        libgl-dev
+        libibus-1.0-dev
+        libltdl-dev
+        libwayland-dev
+        libx11-dev
+        libxext-dev
+        libxft-dev
+        libxkbcommon-dev
+        nasm
```

Comment:
Are we installing these because they all have a distro-independent ABI and we expect the non-dev packages to be installed on all user machines? If so, we may want a comment saying so.